cmd/sovereignnode/systemTestDemo/mockNotifier/disabledGRPCServer.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+type disabledGRPCServer struct {
+}
+
+// NewDisabledGRPCServer -
+func NewDisabledGRPCServer() *disabledGRPCServer {
+	return &disabledGRPCServer{}
+}
+
+// Stop does nothing
```

Comment:
Perhaps make comments consistent by either putting `-` everywhere or actually adding comments

Reply:
Applies to `cmd/sovereignnode/systemTestDemo/mockNotifier/disabledMockGRPCServer.go` as well